python/Ganga/new_tests/GPI/TestJob.py

```diff
@@ -100,4 +105,15 @@ def testJobCopy(self):
         self.assertEqual( j2.postprocessors[0].searchStrings, ["my search"] )
         self.assertTrue( isType(j2.splitter, GenericSplitter) )
         self.assertEqual( j2.splitter.attribute, "application.args" )
-        self.assertEqual( j2.splitter.values, ['arg 1', 'arg 2', 'arg 3'])
\ No newline at end of file
+        self.assertEqual( j2.splitter.values, ['arg 1', 'arg 2', 'arg 3'])
+
+    def test_job_equality(self):
+        """Check that copies of Jobs are equal to each other"""
+        from Ganga.GPI import Job
+        j = Job()
+        j2 = j.copy()
+        j3 = Job(j)
+        assert j == j2
+        assert j2 == j3
+        assert stripProxy(j) == stripProxy(j2)
```

Comment:
Apologies I was thinking of something else when I typed this I think. Ignore the noise